pkg/reconciler/reconciler.go

```diff
@@ -471,6 +486,9 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (res ctrl.Result, err error) {
 	}
 	u.UpdateStatus(updater.EnsureCondition(conditions.Irreconcilable(corev1.ConditionFalse, "", "")))
 
+	// Before we have just releaseHooks, now we have pre and post hooks.
+	// Before the release one shows accepting all: https://helm.sh/docs/topics/charts_hooks/
```

Comment:
Chart hooks are a totally separate concept than than these pre and post hooks. Chart hooks are handled in the helm libraries by the `action.Install`, `action.Upgrade`, and `action.Uninstall` actions. These pre and post hooks are just custom go functions that can be provided when the reconciler is created in `main.go`.